tests/examples/awful/placement/closest_mouse.lua

```diff
@@ -0,0 +1,53 @@
+
+local awful = {placement = require("awful.placement")} --DOC_HIDE
+
+local c = client.gen_fake {x = 20, y = 20, width= 280, height=200, screen =screen[1]} --DOC_HIDE
+
+mouse.coords {x=100,y=100} --DOC_HIDE
+ -- Move the mouse to the closest corner of the focused client
+awful.placement.closest_corner(mouse, {include_sides=true, parent=c})
+mouse.push_history() --DOC_HIDE
```

Comment:
> What's push_history?

The cursor have little lines between the points it visit. That method just create "groups" so unrelated mouse move are not linked. See the other PR, there is a PDF example of this (generated by this example)

> Shouldn't the @foo@-part of the previous commit only be added here?

Maybe, but is it worth it? The code is just eliminated if the variable id not found.

> Wouldn't it be nicer to have "proper" unit tests in spec/ than to have some code whose result isn't really looked at?

I will add some asserts to this one eventually. IMO why having tests in 2 places that are copy/pasted duplicate of each other when you can have only one. Most other example/tests have their asserts, I guess I have been lazy on this one. I will add them.